Sources/MapboxCoreNavigation/Router.swift

```diff
@@ -36,29 +44,29 @@ public protocol Router: CLLocationManagerDelegate {
     /**
      Intializes a new `RouteController`.
      
-     - parameter route: The route to follow.
+     - parameter routeResponse: `RouteResponse` object, containing selection of routes to follow.
      - parameter routeIndex: The index of the route within the original `RouteResponse` object.
      - parameter directions: The Directions object that created `route`.
      - parameter source: The data source for the RouteController.
      - parameter tileStoreLocation: Configuration of `TileStore` location, where Navigation tiles are stored.
      */
-    init(along route: Route, routeIndex: Int, options: RouteOptions, directions: Directions, dataSource source: RouterDataSource, tileStoreLocation: TileStoreConfiguration.Location)
+    init(along routeResponse: RouteResponse, routeIndex: Int, options: RouteOptions, directions: Directions, dataSource source: RouterDataSource, tileStoreLocation: TileStoreConfiguration.Location)
     
     /**
      Details about the user’s progress along the current route, leg, and step.
      */
     var routeProgress: RouteProgress { get }
 
-    /// The route along which the user is expected to travel, plus its index in the `RouteResponse`, if applicable.
-    ///
-    /// If you want to update the route use `Router.updateRoute(with:routeOptions:)` method.
-    var indexedRoute: IndexedRoute { get }
-
     /// The route along which the user is expected to travel.
     ///
     /// You can update the route using `Router.updateRoute(with:routeOptions:)`.
     var route: Route { get }
     
+    /// The `RouteResponse` containing the route along which the user is expected to travel, plus its index in this `RouteResponse`, if applicable.
+    ///
+    /// If you want to update the route use `Router.updateRoute(with:routeOptions:)` method.
+    var indexedRouteResponse: IndexedRouteResponse { get }
```

Comment:
If it isn’t valid to set `indexedRouteResponse` directly, let’s make it publicly read-only.